developer_guides/topology/topology.rst

```diff
@@ -561,7 +561,44 @@ The graph below shows the topology defined in Section 3.1.
 
 .. image:: images/tplg2.png
 
-3. Acronyms
+3. Debugging topology
```

Comment:
If you don't want to care about numbers in list you can use `#.` instead of `1.` `2.` etc